src/TensorKitManifolds.jl

```diff
@@ -28,50 +28,51 @@ checkbase(x, y, z, args...) = checkbase(checkbase(x, y), z, args...)
 # the machine epsilon for the elements of an object X, name inspired from eltype
 eleps(X) = eps(real(eltype(X)))
```

Comment:
Should we make this scaleps? Should we move this to TensorKit?

Reply:
Perhaps, but I would prefer `scalareps`. Which is also just much better than `eleps`.